Formula/netclient.rb

```diff
@@ -0,0 +1,27 @@
+class Netclient < Formula
+  desc "Automate fast, secure, and distributed virtual networks using WireGuard"
+  homepage "https://netmaker.io"
+  url "https://github.com/gravitl/netmaker/archive/v0.15.0.tar.gz"
+  sha256 "cd6f595365dd409400a61b6e7220133d0ea281db1ecc38ac541bc929f748d53e"
+  license "SSPL-1.0"
```

Comment:
SSPL is not an open-source license. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae, https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=915537#15.

Apologies for not spotting this sooner; we should make `brew audit` catch that.

If you wish, you can host this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/